Calendars/Calendars.Plugin.Android/CalendarsImplementation.cs

```diff
@@ -132,7 +131,8 @@ public async Task<IList<CalendarEvent>> GetEventsAsync(Calendar calendar, DateTi
                 throw new ArgumentException("Specified calendar not found on device");
             }
 
-            var eventsUriBuilder = CalendarContract.Instances.ContentUri.BuildUpon();
+            var eventsUriBuilder = CalendarContract.Instances.ContentUri?.BuildUpon()
+                    ?? throw new NullReferenceException("CalendarContract.Instances.ContentUri");
```

Comment:
`NullReferenceException` wouldn't have any message with `!.`. 
I think `BuildUpon` is null only if `CalendarContract.Instances.ContentUri` is null. Wil need to double check that.

Reply:
But yeah, maybe `!.` is cleaner in this case. Removed NRE in favor of `!.`

Reply:
> I think BuildUpon is null only if CalendarContract.Instances.ContentUri is null

You may be right in practice, but judging from the way the API is currently annotated it claims that `BuildUpon` can itself return `null`, separately from whether `ContentUri` is null... as noted in the other comment though, it seems like _everything_ is annotated as possibly returning `null`, so that could be a lie...